syntax = "proto3";

import "empty.proto";
import "table_id.proto";

service CartesianProductCreator {
    rpc create (CartesianProductReguest) returns (Empty) {}
}

message CartesianProductReguest {
    TableId firstId = 1;
    TableId secondId = 2;
}